packages/hawtio/src/plugins/logs/Logs.tsx

```diff
@@ -454,9 +454,10 @@ const LogModal: React.FunctionComponent<{
   )
 }
 
-const LogLevel: React.FunctionComponent<{ level: string }> = ({ level }) => {
+export const LogLevel: React.FunctionComponent<{ level: string }> = ({ level }) => {
```

Comment:
I don't think it's good idea to make a plugin depend on another too much. That should introduce tight couplings between plugins. For such small component, you can just replicate the definition in `Loggers.tsx`.

Reply:
If we find there are a lot more occurences of a similar component in other plugins, we can then consider sharing it in shared.